examples/fabric/v4/network/README.md

```diff
@@ -0,0 +1,34 @@
+# ECX Fabric network CRUD operations
```

Comment:
We're no longer adding to the examples in the Provider repo. Add example usage with values from tfvars.example directly on the resource to the documentation.

Additionally, docs/ need to be added for the resource and data source with including example usage. You can use this doc as a reference for example usage: https://github.com/equinix/terraform-provider-equinix/blob/main/docs/resources/equinix_ecx_l2_connection.md

The addition to examples directory is not needed.